### DISCUSSION TITLE
[question] import Plot in Node.js ERR_REQUIRE_ESM

### DISCUSSION THREAD

**Discussion by dawadam** [observablehq/plot#2066] [Q&A]
Hi, thanks for the great library.
Is it possible to use it only with Node JS?
I try to import like this:

`import Plot from '@observablehq/plot'`

But the error is:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module [...] not supported.

Instead change the require of [...] to a dynamic import() which is available in all CommonJS modules.

```

Why is normal import not working?

**Comment by Fil** [comment]
Did you follow the documentation?
https://observablehq.com/plot/getting-started#installing-from-npm

> npm install @observablehq/plot

You can then load Plot into your app as:

> import * as Plot from "@observablehq/plot";

**Reply by dawadam** [reply]
Yes, I read a lot the documentation.
This is the first method I tried.
This works on the client side, using vue.js, but not on the server side using node.js
I also use TypeScript, maybe an error comes from there.

**Reply by mbostock** [reply]
Please read this guide on ES modules: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

You probably need to add `"type": "module"` to your `package.json` file.
